### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by stephnna**
Hi code reviewer,

I implemented the following:

Create a new Rails app called 'hello-rails-react'.
Use the command rails new -j webpack my_new_rails_app as described in the previous lesson's tutorial.
Verify that the jsbundling-rails gem is added to your Gemfile.
Initialize your project with Git.
Install needed node dependencies as described in the previous lesson's tutorial.
Create a babel.config.js in the root folder of your project as described in the previous lesson's tutorial.
Update webpack.config.js to include the babel loader as described in the previous lesson's tutorial.
Make sure that your project has a Postgres database set up.
Create a table for storing your messages and create 5 different greetings.
Create an API endpoint that selects a random greeting from your table (you will need a controller with an action and Rails route).
Create a static view that will be the root of your app.
Create the App component with react-router and render it in your static view.
Create the Greeting component that will display a greeting. Set it up as a route in your App component.
Create a store, an action, and a reducer that will connect to your API endpoint to get the random greeting.
Display the random greeting in your Greeting component.

Kindly review as necessary
Thank you!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,214 @@
...
+  bundle install
+```
+--->
+
+### Usage
+
+To run the project, execute the following command:
+
+- Run `rails s` or `rails server` to start a development server.
+
```
Comment (brytebee) [inline README.md:134]: - [x] **Required:** Kindly update this section with the proper usage commands.

File: README.md [hunk 2]
```diff
@@ -1 +1,214 @@
...
+#### Local (Terminal)
+
+```sh
+  git clone git@github.com:stephnna/hello-rails-react.git
+  cd hello-rails-react
+```
+
+### Install
+
+Install this project with:
```
Comment (brytebee) [inline README.md:121]: - [x] **Required:** Kindly add the steps to fully clone and install this project for use.

**Review by brytebee — CHANGES_REQUESTED** [review]
Hi, @stephnna👋,

##

### Highlights💡
- Good commit message.✔️ 
- Good PR Summary.✔️
- Great UI design.✔️
- App functions as it should.✔️

##

Good job so far coming so far with the progress you've made! 💪💪
 
However, there are some issues that you still need to work on to go to the next project but you are almost there!



# Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Please, remember to tag me in your question so I can receive the notification.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @brytebee in your question so I can receive the notification.
### _**Use @brytebee on slack for faster responses.**_


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (stephnna) [reply README.md:134]: @brytebee,

Thank you for your review, I don't understand what you mean by '...proper usage commands' The app is actually run by using 'rails s' if you meant another thing please communicate if not kindly approve my project
Thanks!

File: README.md [hunk 3]
```diff
@@ -1 +1,212 @@
...
+  cd budget-app
+  bundle install
+```
+-
+
+### Usage
+
+To run the project, execute the following command:
+
+- Run `rails s` or `rails server` to start a development server.
```
Comment (FrederickMih) [inline README.md:133]: - [Optional]  @stephnna I think `bundle install`, `rails s` are not enough to set up this app completely locally. DB needs to be equally set up.

**Review by FrederickMih — APPROVED** [review]
## Hi 🙋 @stephnna  , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights </img>


- The two apps work together 👍 
- PR is professional ✔️ 
- Linters checks passed ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**